mavros_msgs/Waypoint[] waypoints
---
bool value

